app/src/main/java/com/khinthirisoe/bakingapp/data/db/DbConstant.kt

```diff
@@ -0,0 +1,7 @@
+package com.khinthirisoe.bakingapp.data.db
+
+object DbConstant {
```

Comment:
these constants can be used in other places in addition to db. So, declare then in folder that is meant to be shared across the app

Reply:
I don't get it. hee hee

Reply:
@KhinThiriSoe Do you understand it now? I have explained about this